website/docs/provisioners/connection.html.markdown

```diff
@@ -51,8 +51,7 @@ provisioner "file" {
 
 **The following arguments are supported by all connection types:**
 
-* `type` - The connection type that should be used. Valid types are `ssh` and `winrm`
-  Defaults to `ssh`.
+* `type` - The connection type that should be used. Valid types are `ssh` and `winrm`. Defaults to `ssh`.
```

Comment:
Do you mind keeping the second sentence on the second line?
This is merely to keep it easier to read and maintain.

Reply:
Sure. Sorry for the delayed response.